index.bs

```diff
@@ -40,7 +40,7 @@ table td, table th {
 
 The API defined in this document captures images from a photographic device referenced through a valid {{MediaStreamTrack}}. The produced image can be in the form of a {{Blob}} (see {{takePhoto()}} method) or as a {{ImageBitmap}} (see {{grabFrame()}}). Picture-specific settings can be optionally provided as arguments that can be applied to the device for the capture.
 
-# Image Capture API # {#imagecaptureapi"}
+# Image Capture API # {#imagecaptureapi}
```

Comment:
You need an explanaition up here to explain where the constraints fit in. Something like:

"This document defines a number of new constraints for a MediaStreamTrack that can be applied in order to make its behavior more suitable for taking pictures. The use of these constraints is via the standard getUserMedia() and applyConstraints() methods for a MediaStreamTrack. They will modify the behavior of the track and its source independently of the usage of the track with the ImageCapture object."